### PR TITLE
172 fix save new dashboard name

### PR DESCRIPTION
**Pull request by alison985**
#172

### CODE REVIEW

File: client/app/pages/dashboards/edit-dashboard-dialog.html [hunk 1]
```diff
@@ -5,7 +5,7 @@ <h4 class="modal-title">Edit: {{$ctrl.dashboard.name}}</h4>
 <div class="modal-body">
   <p>
     <form name="myForm">
-      <input name="dashboardName" type="text" class="form-control" placeholder="Dashboard Name" ng-model="$ctrl.dashboard.name" required>
+      <input name="dashboardName" type="text" class="form-control" placeholder="Dashboard Name" ng-model="$ctrl.dashboard.name" ng-keyup="$event.keyCode == 13 && $ctrl.saveDashboard()" required>
```
Comment (spasovski) [inline client/app/pages/dashboards/edit-dashboard-dialog.html]: This seems a bit hacky - I'd rather it call a function that does this check and also comment that keycode 13 is 'enter' for example. Good catch though!

Comment (alison985) [inline client/app/pages/dashboards/edit-dashboard-dialog.html]: Great feedback, very helpful to me. Thanks. Updated.

**Comment by spasovski** [comment]
Awesome thanks!

**Comment by vcarciu** [comment]
Verified as fixed : https://www.screencast.com/t/FYc6pVu6Osr

**Comment by vcarciu** [comment]
I just observed that I misunderstood this issue. Pressing enter will not save the dashboard. Is this already on Stage?

**Comment by alison985** [comment]
@vcarciu This is working for me on staging.

STR:
1. Select "New Dashboard" in the Dashboard drop-down.
2. enter a dashboard name.
3. click Enter on the keyboard.

Expected result:
Dashboard is saved with the name you gave it.

Fail state:
Dashboard does not save on clicking "Enter" key on the keyboard.

**Comment by vcarciu** [comment]
@alison985 

I performed exactly same steps on Friday and I was not able to save by pressing Enter. I don't know what happened, but today it is working : https://www.screencast.com/t/iJGOe8gAm

Verified as fixed

**Pull request merged**